### DISCUSSION TITLE
static file not rendering in deployed version

### DISCUSSION THREAD

**Discussion by Rajesh01-star** [vercel/community#2576] [Help]
### Summary

my project is a simple node page with a database and few html files and some static files (images/css files)
the project works fine in the localhost but in the deployed version the static files are not rendering 

{
    "version":2,
    "builds":[
        {
            "src":"./index.js",
            "use":"@vercel/node"
        }
    ],
    "routes":[
        {
            "src":"/(.*)",
            "dest":"/"
        }
    ],
    "public":"src"
}

this is my vercel.json file 
please help

**Comment by z4nr34l** [comment]
Could you bring us more details?
I consider as helpful those informations:
1. Repository Access;
2. Vercel build logs (from deployment with this issue);
3. Runtime logs from problematic request;

The more - the better for receiving quick help.

You can also try to reproduce this problem on free [CodeSandbox](https://codesandbox.io/) and post URL here when it's done.
